notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java

```diff
@@ -0,0 +1,20 @@
+package com.qpay.notificationmanager.service.impl;
+
+import com.qpay.libs.models.PaymentNotification;
+import com.qpay.notificationmanager.service.NotificationService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
```

Comment:
you don't need it

Reply:
but without it i couldn't initialize service

Reply:
I mean, you could use no args constructor to show you don't have any fields. but you can leave this one

Reply:
ok